### PR TITLE
lib/opensusebasetest.pm: Handle gnome-activities in wait_boot_past_bootloader

### PR DESCRIPTION
**Pull request by Vogtinator**
Missing in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12883.

Verification run: https://openqa.opensuse.org/tests/1847753

### CODE REVIEW

File: lib/opensusebasetest.pm [hunk 1]
```diff
@@ -1050,7 +1051,10 @@ sub wait_boot_past_bootloader {
     # Starting with GNOME 40, upon login, the activities screen is open (assuming the
     # user will want to start something. For openQA, we simply press 'esc' to close
     # it again and really end up on the desktop
-    send_key('esc') if match_has_tag('gnome-activities');
+    if (match_has_tag('gnome-activities')) {
+        send_key 'esc';
+        check_screen 'generic-desktop', $check_interval;
```
Comment (DimStar77) [inline lib/opensusebasetest.pm:1056]: as you already noted: in case we close activities with opensuse-welcome open, we fail here now

Comment (Vogtinator) [reply lib/opensusebasetest.pm:1056]: Needs some perl fu to filter out `gnome-activities` from `@tags`

Comment (DimStar77) [reply lib/opensusebasetest.pm:1056]: `@tags = grep {!/gnome-activites/} @tags;`

**Comment by DimStar77** [comment]
Moved to #12924 - (as Fabian had to leave)

**Pull request closed without merging**